ballista/rust/executor/src/flight_service.rs

```diff
@@ -218,8 +218,8 @@ where
         let batch_flight_data: Vec<_> = batch
             .map(|b| create_flight_iter(&b, &options).collect())
             .map_err(|e| from_arrow_err(&e))?;
-        for batch in &batch_flight_data {
-            send_response(&tx, batch.clone()).await?;
+        for batch in batch_flight_data.into_iter() {
```

Comment:
I don't know why but this was causing a compiler error for me, so I fixed it to get the tests to all pass cleanly on my machine